examples/runners/wgpu/src/compute.rs

```diff
@@ -69,24 +74,29 @@ pub fn start(options: &Options) {
         module: &module,
         entry_point: "main_cs",
     });
-
-    let buf = device.create_buffer(&wgpu::BufferDescriptor {
+    let size = (src.len() * std::mem::size_of::<u32>()).try_into().unwrap();
+    let staging_buffer = device.create_buffer(&wgpu::BufferDescriptor {
         label: None,
-        size: 1,
-        usage: wgpu::BufferUsage::STORAGE,
+        size,
+        // Can be read to the CPU, and can be copied from the shader's storage buffer
+        usage: wgpu::BufferUsage::MAP_READ | wgpu::BufferUsage::COPY_DST,
         mapped_at_creation: false,
     });
 
+    let storage_buffer = device.create_buffer_init(&wgpu::util::BufferInitDescriptor {
+        label: Some("Collatz Conjecture Input"),
+        contents: unsafe { core::slice::from_raw_parts(src.as_ptr().cast::<u8>(), size as usize) },
```

Comment:
I would be fine with bringing in `bytemuck` for this. Should be able to `cast_slice` safely.

Reply:
I've converted it to a safe version which uses `u32::to_ne_bytes` in 01b6076715e000b3efc65ef2834200f485929eca